pkg/start/start.go

```diff
@@ -337,6 +337,7 @@ func createResourceLock(cb *ClientBuilder, namespace, name string) (*resourceloc
 
 func resyncPeriod(minResyncPeriod time.Duration) func() time.Duration {
 	return func() time.Duration {
+		rand.Seed(time.Now().UnixNano())
```

Comment:
seeding makes sense to me.  Can we put this in `cmd/main.go`'s `init` or something, so it only gets called once per CVO launch?

Reply:
Fixed in https://github.com/openshift/cluster-version-operator/commit/a8a347840c4b313d6bf0dc6fe6bca1d1be2fee0a